go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/engelmi/terraform-provider-bluechi
```

Comment:
Not sure at which stage it should happen, but this needs to be changed to `github.com/eclipse-bluechi/terraform-provider-bluechi`

Reply:
Better sooner than later, I think, changed.